code/style.css

```diff
@@ -1,5 +1,208 @@
+.hero {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+
+  background: url(./assets/opera.jpg);
+  min-height: 100vh;
+  background-size: cover;
+  background-position: center;
+}
+
+logo {
+  width: 40px;
+  margin-left: 10px;
+  margin-top: 3px;
+}
+
+img {
+  width: 100%;
+  background-color: white;
+}
+
+/* navigation with hamburger-layer for mobile phone*/
+
+.hamburger-layer {
+  width: 20px;
+  height: 5px;
+  background-color: white;
+  margin-bottom: 2px;
+  margin-right: 10px;
+  margin-top: 3px;
+}
+
+
+.header-text {
+  padding-top: 25%;
+  text-align: center;
+}
+
+
+/* texts of the header for mobile phones */
+h1 {
+  color: #FFDF00;
+  font-size: 40px;
+}
+
+h2 {
+  color: #FFDF00;
+  margin-top: 15%;
+}
+
+/* header for Desktop */
+@media (min-width: 1024px) {
+  .header-text {
+    padding-top: 10%;
+    text-align: center;
+    max-width: 600px;
+  }
+
+  h1 {
+    color: #FFDF00;
+    font-size: 60px;
+  }
+
+  h2 {
+    color: #FFDF00;
+    font-size: 40px;
+    margin-top: 15%;
+  }
+}
+
+
+/* Title of form for mobile phones */
+
+h3 {
+  color: rgb(81, 81, 132);
+  font-weight: 200;
+  padding-left: 5%;
+}
+
+/* Title of form for Desktop */
+@media (min-width: 1024px) {
+  h3 {
+    font-weight: 500;
+    text-align: left;
+    font-size: xx-large;
+  }
+}
+
+/* sign-up form for mobile phones */
+.form-container {
+  display: flex;
+  justify-content: center;
+  background-color: rgb(236, 233, 233);
+  padding: 5%;
+  border-radius: 5px;
+  margin: auto;
+  max-width: 500px;
+}
+
+form {
+  display: flex;
+  width: 80%;
+  display: block;
+  justify-content: center;
+}
+
+
+/* personal information */
+
+label {
+  display: block;
+  text-align: left;
+  padding: 5px;
+}
+
+input {
+  width: 100%;
+  justify-content: center;
+  border: 2px solid rgb(200, 200, 200);
+  border-radius: 5px;
+}
+
+/* Inputs for Tablets, iPads, desktop */
+@media (min-width: 667px) {
+  input {
+    width: 300px;
+  }
+}
+
+/* Input for Desktop */
+@media (min-width:1024px) {
+  input:hover {
+    background-color: rgb(219, 218, 218);
+  }
+}
+
+/* country: drop-down menu */
+.drop-down {
+  border: 2px solid rgb(200, 200, 200);
+  border-radius: 5px;
+}
+
+/* terms and conditions: checkbox-style */
+.terms-and-conditions {
+  display: flex;
+}
+
+#checkbox {
+  width: 20px;
+}
+
+/* submit-button */
+
+#submit-button-container {
+  display: flex;
+  width: 80%;
+  margin-left: 10%;
+  margin-top: 5px;
+  justify-content: center;
+  background-color: rgb(166, 166, 188);
+  border-radius: 5px;
+  padding: 5px;
+}
+
+#submit-button {
+  background-color: rgb(166, 166, 188);
+  border: none;
+  width: 100%;
+}
+
+#submit-button:active {
+  background-color: #673dda;
+}
+
+/* Input for Desktop */
+@media (min-width:1024px) {
+  #submit-button:hover {
+    background-color: rgb(83, 53, 171);
+    color: white;
+  }
+
+
+}
+
+/* footer for mobile phone */
+footer {
+  display: flex;
+  display: row;
```

Comment:
display: row;  isn't a valid value. Correct way to set row layout (default for flex): flex-direction: row;